packages/react-components/react-nav-preview/src/components/NavDrawer/useNavDrawer.ts

```diff
@@ -13,11 +14,20 @@ import type { NavDrawerProps, NavDrawerState } from './NavDrawer.types';
  * @param ref - reference to root HTMLDivElement of NavDrawer
  */
 export const useNavDrawer_unstable = (props: NavDrawerProps, ref: React.Ref<HTMLDivElement>): NavDrawerState => {
-  const inlineDrawerState = useInlineDrawer_unstable(props, ref);
-  const navState = useNav_unstable(props, ref);
+  const focusAttributes = useArrowNavigationGroup({ axis: 'vertical', circular: true });
+  const baseDrawerState = useInlineDrawer_unstable(props, ref);
+  const navState = useNav_unstable(
+    {
+      role: 'navigation',
```

Comment:
It could be - for now I don't think I'm going to invest much more in the Nav component, I want most folks to consume the NavDrawer. I worry that redundant 'navigation' roles will be a trip hazard.